lib/createChangelog.js

```diff
@@ -1,23 +1,45 @@
 import moment from 'moment';
 import R from 'ramda';
 
-function createChangelog(newVersionNumber, validLabels, mergedPullRequests) {
-    const groupedPullRequests = R.groupBy(R.prop('label'), mergedPullRequests);
-    const date = moment().locale('en').format('MMMM D, YYYY');
-    const title = `## ${newVersionNumber} (${date})`;
+const COLLAPSE_COUNT = 10;
 
-    let changelog = `${title}\n\n`;
+function formatPR(pr) {
+    return `-   ${pr.title} [#${pr.id}](https://github.com/storybooks/storybook/pull/${pr.id})\n`;
```

Comment:
👍 for providing a link to the PR, but we should determine the github owner and repo automatically.